.travis.yml

```diff
@@ -1,4 +1,5 @@
-sudo: false
+dist: trusty
+sudo: required
```

Comment:
Can we add a comment explaining why we have the `sudo: required` line? IIUC, it's not because we need sudo features, but, it's to get the trusty image.